test/spec/hasFocus.js

```diff
@@ -0,0 +1,9 @@
+describe('hasFocus', () => {
+    it('should return true if element is active', async function () {
+        (await this.client.hasFocus('[name="login"]')).should.be.true
+    })
+
+    it('should return false on inactive elements', async function () {
+        (await this.client.hasFocus('body')).should.be.false
+    })
+})
```

Comment:
this file should be located in `test/spec/desktop/` as it only can be executed with a browser